admin/plugins.php

```diff
@@ -0,0 +1,411 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets plugin info from plugin slug.
```

Comment:
Worth clarifying that this includes a remote request, e.g.:
```suggestion
 * Gets plugin info for the given plugin slug from WordPress.org.
```